src/View/CharacterDetail/CharacterDetail.test.jsx

```diff
@@ -0,0 +1,33 @@
+import { screen, render } from '@testing-library/react';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+import CharacterDetail from './CharacterDetail';
+import { mockSingleCharacter } from '../../mocks/TestData';
+import { MemoryRouter } from 'react-router-dom';
+
+const server = setupServer(
+  rest.get('https://api.disneyapi.dev/characters/:characterId', (req, res, ctx) => {
+    return res(ctx.json(mockSingleCharacter));
+  })
+);
+
+beforeAll(() => server.listen());
+afterAll(() => server.close());
+
+test('should render a character image, name, and link', async () => {
+  render(
+    <MemoryRouter initialEntries={['/characters/5']} initialIndex={1}>
```

Comment:
Curious why you didn't need to include the match for it to pass. Maybe because of MSW?
`    <MemoryRouter>
      <CharacterDetail match={{ params: { id: 5 } }} />
    </MemoryRouter>`